frontend/src/components/Main.js

```diff
@@ -1,17 +1,19 @@
-import { Fragment }from 'react'
+import { Fragment, useState }from 'react'
 import SearchForm from './SearchForm';
 import ResultsList from './ResultsList';
 import '../sass/components/main.scss';
 
 function Main() {
-  return <Fragment>
+    const [difficulty, setDifficulty] = useState('beginner');
+    return (
+        <Fragment>
             <main className='main'>
             <h1 className='heading-primary'>abb - always be building</h1>
-
-                <SearchForm />
-                <ResultsList />
+                <SearchForm difficulty={difficulty} setDifficulty={setDifficulty}/>
```

Comment:
It is not good practice to pass the function to set state as a prop to a component. What is industry standard is to create a subscriber function with the logic to set state and pass that function as a prop instead to the component that will change the state. So, 
```javascript
const onSetDifficulty = (value) => {
      setDifficulty(value)
}
```
This is the better practice.